docs/content-editing/add-a-search-block.md

```diff
@@ -0,0 +1,431 @@
+    
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-17.jpeg
+    :alt: Click on `Add Facet` to add a facet.
+    :target: /_images/add-a-search-block-step-17.jpeg
+    ```
+    +++
+    _Click on {guilabel}`Add Facet` to add a facet._
+    ````
+
+1.  Click the Label field to add a label to each facet.
+
+    Type "Tags" as you'll be adding a facet to filter by tag.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-18.jpeg
+    :alt: Click the Label field to add a label to each facet.
+    :target: /_images/add-a-search-block-step-18.jpeg
+    ```
+    +++
+    _Click the Label field to add a label to each facet._
+    ````
+
+1.  Click on {guilabel}`Field` to see the options available.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-19.jpeg
+    :alt: Click on `Field` to see the options available.
+    :target: /_images/add-a-search-block-step-19.jpeg
+    ```
+    +++
+    _Click on {guilabel}`Field` to see the options available._
+    ````
+
+1.  Click on {guilabel}`Tag`.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-20.jpeg
+    :alt: Click on `Tag`.
+    :target: /_images/add-a-search-block-step-20.jpeg
+    ```
+    +++
+    _Click on {guilabel}`Tag`._
+    ````
+
+1.  Click on {guilabel}`Facet widget` to pick which type of widget you want to use for this facet.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-21.jpeg
+    :alt: Click on `Facet widget` to pick which type of widget you want to use for this facet.
+    :target: /_images/add-a-search-block-step-21.jpeg
+    ```
+    +++
+    _Click on {guilabel}`Facet widget` to pick which type of widget you want to use for this facet._
+    ````
+
+1.  Select {guilabel}`Checkbox` to display a list of available Tags as checkboxes.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-22.jpeg
+    :alt: Select `Checkbox` to display a list of available Tags as checkboxes.
+    :target: /_images/add-a-search-block-step-22.jpeg
+    ```
+    +++
+    _Select {guilabel}`Checkbox` to display a list of available Tags as checkboxes._
+    ````
+
+1.  Click on {guilabel}`Add Facet` to add another facet.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-23.jpeg
+    :alt: Click on `Add Facet` to add another facet.
+    :target: /_images/add-a-search-block-step-23.jpeg
+    ```
+    +++
+    _Click on {guilabel}`Add Facet` to add another facet._
+    ````
+
+1.  Click the Label field to add a label to each facet.
+
+    Type "Date range" as you'll be adding a facet to filter by tag.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-24.jpeg
+    :alt: Click the Label field to add a label to each facet.
+    :target: /_images/add-a-search-block-step-24.jpeg
+    ```
+    +++
+    _Click the Label field to add a label to each facet._
```

Comment:
```suggestion
1.  Click the {guilabel}`Label` field to add a label to each facet.

    Type "Date range" as you'll be adding a facet to filter by tag.

    ````{card}
    ```{image} /content-editing/_static/add-a-search-block-step-24.jpeg
    :alt: Click the `Label` field to add a label to each facet.
    :target: /_images/add-a-search-block-step-24.jpeg
    ```
    +++
    _Click the {guilabel}`Label` field to add a label to each facet._
```